spec/client/ics-007-tendermint-client/README.md

```diff
@@ -59,14 +59,16 @@ This specification depends on correct instantiation of the [Tendermint consensus
 
 ### Client state
 
-The Tendermint client state tracks the current revision, current validator set, trusting period, unbonding period, latest height, latest timestamp (block time), and a possible frozen height.
+The Tendermint client state tracks the current revision, current validator set, trusting period, unbonding period, delayTimePeriod, delayBlockPeriod, latest height, latest timestamp (block time), and a possible frozen height. 
 
 ```typescript
 interface ClientState {
   chainID: string
   trustLevel: Rational
   trustingPeriod: uint64
   unbondingPeriod: uint64
+  delayTimePeriod: uint64 
```

Comment:
Shall we put an upper bound check for delayPeriods in the client initialisation to avoid misconfigurations?